<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>sortWindow</class>
 <widget class="QMainWindow" name="sortWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>308</width>
    <height>345</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QComboBox" name="sort_alogrithms">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>10</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string notr="true">Sorting algorithms</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Insertion Sort</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Merge Sort</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Selection Sort</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Bubble Sort</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Quick Sort</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Counting Sort</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Heap Sort</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Cycle Sort</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Radix Sort</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Shell Sort</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Bucket Sort</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>40</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Ascend / Descend</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Ascending</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Descending</string>
     </property>
    </item>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>70</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Multi sort</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>150</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Searching algorithms</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Linear Search</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Binary Search</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Jump Search</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>100</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Sort</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>230</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Search</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>190</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>308</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
